server/aws-lsp-codewhisperer/package.json

```diff
@@ -21,9 +21,12 @@
         "compile": "tsc --build",
         "postcompile": "npm run copyServiceClient",
         "copyServiceClient": "copyfiles -u 1 --error ./src/client/sigv4/*.json out && copyfiles -u 1 --error ./src/client/token/*.json out",
+        "lint": "npm run lint:src",
+        "lint:bundle:webworker": "webpack --config webpack.lint.config.js && eslint bundle/aws-lsp-codewhisperer-webworker.js # Verify compatibility with web runtime target",
```

Comment:
is `lint:bundle:webworker` supposed to be run only manually?

Reply:
for now yes. Adding it to any monorepo-level scripts will cause too many errors, potentially blocking other developers. For now this is a convenience tool at development time. We can extend this to release time, I may follow up on that.